### ISSUE TITLE
CSS bug: content: " ("

### ISSUE THREAD

**Issue by tomByrer** [xem/miniMinifier#2]
In the Bootstrap.css:

``` CSS
content: " (" attr(href) ")"
```

is compressed as

``` CSS
content:"(" attr(href)")"
```

should be

``` CSS
content:" ("attr(href)")"
```

Otherwise, neat tool(s)!

**Comment by xem** [comment]
Indeed, my CSS & JS minifiers don't handle strings yet, but I'll try to fix that. Thanks!

**Comment by tomByrer** [comment]
Thanks for your consideration!
If you want, I can PR on your[index.html](https://github.com/xem/miniMinifier/blob/gh-pages/css/index.html) to add a warning, unless you plan to fix it soonish?

**Comment by xem** [comment]
I added the warnings in the CSS & JS pages.
I don't know how to fix it yet, though :) please be patient.

**Comment by tomByrer** [comment]
Cheers!
Take your time :)  I just happened to have SO window open; [this could help](http://stackoverflow.com/questions/14540094/javascript-regular-expression-for-removing-all-spaces-except-for-what-between-do)?
BTW, this [REGEX testing tool](https://www.debuggex.com/) is fun :)

**Comment by xem** [comment]
Thanks for those links! That will be handy.
Oh and if you're into regexes, feel free to try and fix the CSS minifier and send a PR ;)
